runtime/convertValues.go

```diff
@@ -731,17 +834,13 @@ func importPublicKey(
 			// This is calculated when creating the public key.
 
 		default:
-			return fmt.Errorf(
+			return nil, fmt.Errorf(
 				"cannot import value of type '%s'. invalid field '%s'",
 				ty,
-				fieldName,
+				field.Name,
 			)
 		}
 
-		return nil
-	})
-	if err != nil {
```

Comment:
The code used to be an iteration function which set an outer error that needed to be checked, but it was replaced with a for-range iteration, so the error can be returned directly from the loop